### ISSUE TITLE
about MACS and FLOPS

### ISSUE THREAD

**Issue by leeqiaogithub** [sovrasov/flops-counter.pytorch#98]
I suspect ptflops confused MACS with FLOPS.

for example:
I got one conv = nn.Conv2d(3, 32, kernel_size=7, stride=2, padding=3)
ptflops output is :  4.91 GMac
(conv): Conv2d(0.005 M, 0.158% Params, **4.91 GMac**, 1.879% MACs, 3, 32, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3))
We know by hand that conv operetor has a force of 4.877GFLPS.【2*k*k*Cin*Cout*Hout*Wout】

So I want to know if the library has the wrong units

**Comment by sovrasov** [comment]
See https://github.com/sovrasov/flops-counter.pytorch/issues/16 and https://github.com/sovrasov/flops-counter.pytorch/blob/master/ptflops/pytorch_ops.py#L53 for the reference. I don't know the input / output resolution in your example, so I can't say if it is correct or not.

**Comment by leeqiaogithub** [comment]
@sovrasov I think I made a mistake. My network was called twice, so the difference was exactly two times.  But thank you for your reply!
